library/CHART-TEMPLATE/README.md

```diff
@@ -36,17 +36,27 @@ low data modes or any other quirk that it could have.
 | Key | Type | Default | Description |
 |-----|------|---------|-------------|
 | affinity | object | `{}` |  |
+| apiKey | string | `"barqux"` |  |
```

Comment:
Is it called userKey in the docs? https://docs.newrelic.com/docs/apis/intro-apis/new-relic-api-keys/

I think that we should use the same name or add a explanatory description

Reply:
Good point. Changed on [#0486574](https://github.com/newrelic/helm-charts/pull/1463/commits/0486574ced2ba833ae6bd823d51ba4bbe2c4a6f5)